lib/objects/objectsInMemServerRedis.js

```diff
@@ -660,13 +681,13 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
                             response.push(this.getFileId(entryId, baseName + arr.file, true));
                             response.push(this.getFileId(entryId, baseName + arr.file, false));
                         });
-                        handler.sendArray(responseId, response);
+                        handler.sendArray(responseId, isScan ? ['0', response] : response);
                     });
                 }
             } else {
-                handler.sendError(responseId, new Error('KEYS-UNSUPPORTED for namespace ' + namespace + ': Data=' + JSON.stringify(data)));
+                handler.sendError(responseId, new Error(`KEYS/SCAN-UNSUPPORTED for namespace ${namespace}: Pattern=${pattern}`));
```

Comment:
Can you please use isScan to output KEY or SCAN at the error message beginning? ;-) It is an implementation detail that we use the same code here, or? ;-)